src/vuex.ts

```diff
@@ -10,10 +11,12 @@ const store = new Store({
   state: new RootState(),
   modules: {
     ...extractVuexModule(ExampleStore),
+    ...extractVuexModule(TodosStore),
   },
 });
 
 const example = createProxy(store, ExampleStore);
+const todos = createProxy(store, TodosStore);
 
 export default store;
-export { example };
\ No newline at end of file
+export { example, todos, Todo };
```

Comment:
you may want to export `Todo` type only in the correct folder, exporting it also here may lead to confusion
I think just `import { Todo } from "@/store/todo"` is far more readable.

Reply:
I agree